CONTRIBUTING.md

```diff
@@ -27,7 +27,11 @@ pipenv shell
 
 py.test graphql_compiler/tests
 ```
-
+If you have any trouble starting the mysql database, make sure mysql service is not already running
```

Comment:
Might be worth making the distinction of "running outside of docker" here, so the user isn't confused about why they are stopping the service only to start it up again.

Reply:
Makes sense